packaging/linux/buildpackage.sh

```diff
@@ -1,41 +1,60 @@
 #!/bin/bash
 # OpenRA packaging master script for linux packages
 
-if [ $# -ne "3" ]; then
-	echo "Usage: `basename $0` tag files-dir outputdir"
+command -v curl >/dev/null 2>&1 || { echo >&2 "Linux packaging requires curl."; exit 1; }
+command -v markdown >/dev/null 2>&1 || { echo >&2 "Linux packaging requires markdown."; exit 1; }
+
+if [ $# -ne "2" ]; then
+	echo "Usage: `basename $0` tag outputdir"
     exit 1
 fi
 
-TAG=$1
-VERSION=`echo $TAG | grep -o "[0-9]\\+-\\?[0-9]\\?"`
-BUILTDIR=$2
-PACKAGEDIR=$3
-ROOTDIR=root
+# Set the working dir to the location of this script
+cd $(dirname $0)
+
+TAG="${1}"
+VERSION=`echo ${TAG} | grep -o "[0-9]\\+-\\?[0-9]\\?"`
+OUTPUTDIR="${2}"
+
+SRCDIR="$(pwd)/../.."
+BUILTDIR="$(pwd)/build"
 
 # Clean up
-rm -rf $ROOTDIR
+rm -rf ${BUILTDIR}
+
+echo "Building core files"
 
-cd ../..
+pushd ${SRCDIR} > /dev/null
+make linux-dependencies
+make core SDK="-sdk:4.5"
+make version VERSION="${TAG}"
 
-# Copy files for OpenRA.Game.exe and OpenRA.Editor.exe as well as all dependencies.
-make install prefix="/usr" DESTDIR="$PWD/packaging/linux/$ROOTDIR"
+make install-core prefix="/usr" DESTDIR="${BUILTDIR}"
+make install-linux-shortcuts prefix="/usr" DESTDIR="${BUILTDIR}"
+make install-linux-mime prefix="/usr" DESTDIR="${BUILTDIR}"
+make install-linux-appdata prefix="/usr" DESTDIR="${BUILTDIR}"
+make install-man-page prefix="/usr" DESTDIR="${BUILTDIR}"
 
-# Install startup scripts, desktop files and icons
-make install-linux-shortcuts prefix="/usr" DESTDIR="$PWD/packaging/linux/$ROOTDIR"
-make install-linux-mime prefix="/usr" DESTDIR="$PWD/packaging/linux/$ROOTDIR"
-make install-linux-appdata prefix="/usr" DESTDIR="$PWD/packaging/linux/$ROOTDIR"
-make install-man-page prefix="/usr" DESTDIR="$PWD/packaging/linux/$ROOTDIR"
+popd > /dev/null
 
 # Documentation
-mkdir -p $PWD/packaging/linux/$ROOTDIR/usr/share/doc/openra/
-cp *.html $PWD/packaging/linux/$ROOTDIR/usr/share/doc/openra/
+DOCSDIR="${BUILTDIR}/usr/share/doc/openra/"
+
+mkdir -p "${DOCSDIR}"
+
+curl -s -L -O https://raw.githubusercontent.com/wiki/OpenRA/OpenRA/Changelog.md
```

Comment:
The wiki is a separate hidden repository, so the best we could do is to require the caller to manually pass in the version hash which then wouldn't work with travis.

My preferred solution would be to not ship the changelog at all.  On all three platforms we bury it in system directories that normal players will never look inside.